package.json

```diff
@@ -24,5 +22,33 @@
   "engines": {
     "node": "12.x",
     "npm": "7.x || 8.x"
+  },
+  "dependencies": {
```

Comment:
**Question:** Is there a reason for these to be dependencies and not peer dependencies?

Reply:
I see, I guess to me (an fresh set of eyes) the distinction wasn't clear. Can you explain why we need the peer dependencies?

Reply:
These were the peerDependencies are required by the [packages in dependencies](https://github.com/Financial-Times/origami/blob/dece7a9391c12478fb9e114821812f8d950ddaba/components/o-quote/package.json#L24).  NPM at the time complained about these missing peerDependencies. I was trying to replicate the error to add the message here, but it seems to be fine without it now. So I've removed the peerDependencies from here.

Reply:
Thanks Bolu, could you also double check on whats the correct approach dependencies or peer dependencies for origami dependencies. I had a feeling we should use peer dependencies as this helps to mimic the dependencies structure of bower.